src/aiortc/rtcrtptransceiver.py

```diff
@@ -79,6 +79,10 @@ def kind(self) -> str:
     def mid(self) -> Optional[str]:
         return self.__mid
 
+    @property
+    def mline_index(self) -> Optional[int]:
```

Comment:
This is the change that is causing me a problem.

https://www.w3.org/TR/webrtc/#webidl-1228768784 => RTCRtpTransceiver does not have an `mline_index` property

Reply:
Aha! I see. Thanks for pointing out.